src/implementations/twig/components/footer-core/footer-core.html.twig

```diff
@@ -97,10 +100,6 @@
         extra_classes: 'ecl-footer-core__logo-link'
       } only %}
     {% endif %}
-    {# content before #}
```

Comment:
this part is not used on EU footer (it was used on EC one, and the code was the same).
I could bring it back, but as the content of the footer can't be modified on a site level (it's corporate), I don't think that it would cause trouble